src/EntityFrameworkCore.Projectables.Abstractions/ProjectableAttribute.cs

```diff
@@ -13,6 +13,16 @@ namespace EntityFrameworkCore.Projectables
     [AttributeUsage(AttributeTargets.Method | AttributeTargets.Property, Inherited = true, AllowMultiple = false)]
     public sealed class ProjectableAttribute : Attribute
     {
+        public ProjectableAttribute() { }
+
+        public ProjectableAttribute(string useMemberBody, object memberBodyParameterValue)
```

Comment:
A constructor would be an aggressive change as this would limit our ability in the future to add additional constructor arguments. Just having a property would be sufficient. Can this constructor be dropped?

Reply:
Comment accepted